.gitignore

```diff
@@ -1,3 +1,3 @@
 .idea
-helpers/helpers.in
-helpers/helpers.out
\ No newline at end of file
+app/helpers/helpers/helpers.in
+app/helpers/helpers/helpers.out
```

Comment:
please add a gitignore entry for https://github.com/TudorHulban/GoLayouter/blob/cezar/textfile  
if this is an output, let us rename it render_for_name of input without extension